app/scripts/directives/editPvc.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .directive("editPvc",
+             function($uibModal,
+                      $filter,
+                      $routeParams,
+                      APIService,
+                      DataService,
+                      ProjectsService,
+                      NotificationsService,
+                      Logger) {
+
+    return {
+      restrict: "E",
+      scope: {
+        // pvc object
+        pvc: "@",
```

Comment:
Why `@`? Shouldn't this be `<` (or possibly `=` if you need two-way binding)?

Reply:
ACK!

the two way binding was not required so I used <.